pandas/core/ops/methods.py

```diff
@@ -171,8 +171,6 @@ def _create_methods(cls, arith_method, comp_method, bool_method, special):
         mul=arith_method(cls, operator.mul, special),
         truediv=arith_method(cls, operator.truediv, special),
         floordiv=arith_method(cls, operator.floordiv, special),
-        # Causes a floating point exception in the tests when numexpr enabled,
-        # so for now no speedup
```

Comment:
This message should have been moved in #19649

I think Ok if we have not had reports of floating point exceptions since enabled.